### PR TITLE
Adds GoogleNewsBundle

### PR DESCRIPTION
**Pull request by htollefsen**
Solves #8.

### CODE REVIEW

**Comment by htollefsen** [comment]
Tests wont pass untill https://github.com/transfer-framework/ezplatform/pull/79 is solved.

File: ezpublish/googlenewsbundle/src/Transfer/GoogleNewsBundle/Transfer/Manifest/GoogleNewsToEzPlatformContentManifest.php [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                ->createProcedure('google_news_contenttype')
+
+                    ->addSource(new LocalDirectoryAdapter(array('directory' => __DIR__.'/../../Resources/contenttypes')))
+
+                        ->addWorker(function (ValueObject $object) { return $object->data; })
+                        ->addWorker(new YamlToArrayTransformer())
+                        ->addWorker(new ArrayToEzPlatformContentTypeObjectTransformer())
+                        ->addWorker(new SplitterWorker())
+
+            ->addWorker(function (ValueObject $object) {
```
Comment (valisj) [inline ezpublish/googlenewsbundle/src/Transfer/GoogleNewsBundle/Transfer/Manifest/GoogleNewsToEzPlatformContentManifest.php]: Something funky happened here.

File: ezpublish/googlenewsbundle/src/Transfer/GoogleNewsBundle/Transfer/Worker/XmlToArrayTransformer.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?php
```
Comment (valisj) [inline ezpublish/googlenewsbundle/src/Transfer/GoogleNewsBundle/Transfer/Worker/XmlToArrayTransformer.php]: Should we try incorporating existing workers, such as: https://github.com/transfer-framework/commons/blob/1.0/src/Transfer/Commons/Xml/Worker/Transformer/StringToSimpleXmlTransformer.php?

File: ezpublish/googlenewsbundle/composer.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      "name": "Harald Tollefsen",
+      "email": "harald@tollefsen.info"
+    }
+  ],
+  "require": {
+    "php": "~5.5|~7.0",
+    "symfony/yaml": "~2.7",
+    "monolog/monolog": "~1.17",
+    "doctrine/orm": "^2.5",
+    "transfer/ezplatform": "1.0.x-dev"
```
Comment (valisj) [inline ezpublish/googlenewsbundle/composer.json]: Are you sure there won't be any BC breaks, if we leave this constraint as is?

File: ezpublish/googlenewsbundle/tests/bootstrap.php [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    if (!symlink($configDir.'/config.php-DEVELOPMENT', $configDir.'/config.php')) {
+        throw new \RuntimeException('Could not symlink config.php-DEVELOPMENT to config.php!');
+    }
+}
+
+/** @var Composer\Autoload\ClassLoader $loader */
+$loader = require __DIR__.'/../vendor/autoload.php';
+
+/** 
+/unit/Transfer/Tests
```
Comment (valisj) [inline ezpublish/googlenewsbundle/tests/bootstrap.php]: Do you why this is has a red background?

File: ezpublish/googlenewsbundle/composer.json (hunk 3 above)
Comment (htollefsen) [inline ezpublish/googlenewsbundle/composer.json]: I could/should have pointed this toward the beta 2maybe. On the the other hand I think the examples should be working towards the latest dev-release until a stable release of transfer/ezplatform is out

File: ezpublish/googlenewsbundle/src/Transfer/GoogleNewsBundle/Transfer/Manifest/GoogleNewsToEzPlatformContentManifest.php (hunk 1 above)
Comment (htollefsen) [inline ezpublish/googlenewsbundle/src/Transfer/GoogleNewsBundle/Transfer/Manifest/GoogleNewsToEzPlatformContentManifest.php]: I'll see if I can make funk go away tomorrow

File: ezpublish/googlenewsbundle/src/Transfer/GoogleNewsBundle/Transfer/Worker/XmlToArrayTransformer.php (hunk 2 above)
Comment (htollefsen) [inline ezpublish/googlenewsbundle/src/Transfer/GoogleNewsBundle/Transfer/Worker/XmlToArrayTransformer.php]: Yeah, I didn't think about the existing transfer/common. I'll look for a solution

File: ezpublish/googlenewsbundle/tests/bootstrap.php (hunk 4 above)
Comment (htollefsen) [inline ezpublish/googlenewsbundle/tests/bootstrap.php]: I don't really know,  I'll see if SO can help

**Comment by htollefsen** [comment]
Anything else you think should be added/is missing @valise?

**Comment by valisj** [comment]
@htollefsen Didn't notice that the issues were fixed until now. Great work!

**Pull request merged**